functions/daemon.go

```diff
@@ -187,6 +194,30 @@ func startGoRoutines(wg *sync.WaitGroup) context.CancelFunc {
 	go Checkin(ctx, wg)
 	wg.Add(1)
 	go networking.StartIfaceDetection(ctx, wg, config.Netclient().ListenPort)
+
+	// MQTT Fallback Channel and Ticker
+	MQFallbackStop = make(chan bool)
+	MQFallbackTicker = time.NewTicker(MQ_FALLBACK_TIMEOUT)
+	MQFallbackTickerStop = make(chan bool)
+
+	// MQTT Peer Update fallback ticker
+	go func() {
```

Comment:
move this into a function and use the wait group to exit safely when main thread stops